doc/releases/changelog-0.9.0.md

```diff
@@ -210,127 +226,140 @@
+  ```python
+  qml.device("null.qubit", wires=1)
 
-  A new MLIR pass `detensorize-scf` is added that works in conjunction with the existing
-  `linalg-detensorize` pass to detensorize input programs. The IR generated by JAX wraps all values
-  in the program in tensors, including scalars, leading to unnecessary memory allocations for
-  programs compiled to CPU via the MLIR-to-LLVM pipeline.
+  @qml.qjit
+  @qml.qnode(dev)
+  def g(x):
+      qml.RX(x, wires=0)
+      return qml.probs(wires=[0])
+  ```
 
-* Bufferization of `gradient.ForwardOp` and `gradient.ReverseOp` now requires three steps:
-  `gradient-preprocessing`, `gradient-bufferize`, and `gradient-postprocessing`.
-  `gradient-bufferize` has a new rewrite for `gradient.ReturnOp`.
-  [(#1139)](https://github.com/PennyLaneAI/catalyst/pull/1139)
+* Setting the `seed` argument in the `qjit` decorator will now seed sampled results, in addition
+  to mid-circuit measurement results.
+  [(#1164)](https://github.com/PennyLaneAI/catalyst/pull/1164)
 
-* The decorator `self_inverses` now supports all Hermitian Gates.
-  [(#1136)](https://github.com/PennyLaneAI/catalyst/pull/1136)
+  ```python
+  dev = qml.device("lightning.qubit", wires=1, shots=10)
 
-  The full list of supported gates are as follows:
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      m = catalyst.measure(0)
 
-  One-qubit Gates:
-  - [`qml.Hadamard`](https://docs.pennylane.ai/en/stable/code/api/pennylane.Hadamard.html)
-  - [`qml.PauliX`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliX.html)
-  - [`qml.PauliY`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliY.html)
-  - [`qml.PauliZ`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliZ.html)
+      if m:
+          qml.Hadamard(0)
 
-  Two-qubit Gates:
-  - [`qml.CNOT`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CNOT.html)
-  - [`qml.CY`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CY.html)
-  - [`qml.CZ`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CZ.html)
-  - [`qml.SWAP`](https://docs.pennylane.ai/en/stable/code/api/pennylane.SWAP.html)
+      return qml.sample()
 
-  Three-qubit Gates:
-  - [`qml.Toffoli`](https://docs.pennylane.ai/en/stable/code/api/pennylane.Toffoli.html)
+  @qml.qjit(seed=37, autograph=True)
+  def workflow(x):
+      return jnp.squeeze(jnp.stack([circuit(x) for i in range(4)]))
+  ```
+  ```pycon
+  >>> workflow(1.8)
+  Array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+       [1, 1, 0, 0, 1, 1, 0, 0, 1, 0],
+       [0, 0, 1, 0, 1, 1, 0, 0, 1, 1],
+       [1, 1, 1, 0, 0, 1, 1, 0, 1, 1]], dtype=int64)
+  >>> workflow(1.8)
+  Array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [1, 1, 0, 0, 1, 1, 0, 0, 1, 0],
+         [0, 0, 1, 0, 1, 1, 0, 0, 1, 1],
+         [1, 1, 1, 0, 0, 1, 1, 0, 1, 1]], dtype=int64)
+  ```
+
+  Note that statistical measurement processes such as `expval`, `var`, and `probs`
+  are currently not affected by seeding when shot noise is present.
```

Comment:
@paul0403 this caught me off-guard/feels like a bit of a bug. I was expecting `expval(...)` to _also_ be seeded in the presence of shot noise, but it looks like it still fluctuates?

Reply:
While this is true, I would say this isn't actually a bug, since **the seeding feature was never meant to be user facing** and is only supposed to be used internally to combat CI flaky tests. Thus as long as we document which circuit readout methods are seeded and which aren't, I think we are good.

Reply:
This is very much user facing, however, and I can see it being a valuable feature to end users 😎

So let's treat it as a feature that we will plan to extend to other measurement processes

Reply:
I see, how about we keep a list of seeding-supported circuit readouts on the documentation for now, and make plans to seed more readouts in the future?

Reply:
#1248